### PR TITLE
CVS-78189/65104/78311 Documentation adjustments for 2022.1

### PR DESCRIPTION
**Pull request by dkalinowski**

### CODE REVIEW

**Review by atobiszei — APPROVED** [review]

**Comment by dkalinowski** [comment]
Document existence of `PERFORMANCE_HINT` parameter for MULTI/AUTO

File: docs/demultiplexing.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ Example configuration file with one demultiplexer (remove everything after arrow
...
+There might be use cases where one custom node library is used to produce an unpredictable number of a batch. To achieve it, `demultiply_count` can be set to `-1`. This indicates that the pipeline supports any number of batch returned by custom node: `(X,N,C,H,W,...)` - where `X` is dynamic `demultiply_count`. OpenVINO&trade; Model Server is capable of interpreting such dynamic batch and is able to split outputs into a dynamic number of the pipeline branches. When using dynamic `demultiply_count` parameters, only one demultiplexer can exist in the pipeline. Important to note - in release 2021.3, when batch 0 is returned, the pipeline stops its execution and ABORTED status is returned. This may be changed in future releases.
```
Comment (mzegla) [inline docs/demultiplexing.md:75]: Looks like we break compatibility with previous configs. Please make sure we mention that in release notes.

**Review by mzegla — APPROVED** [review]

File: docs/demultiplexing.md (hunk 1 above)
Comment (dkalinowski) [reply docs/demultiplexing.md:75]: 0 is still supported, but deprecated in 2022.1. We plan on removing it in 2022.2.
This is included in release notes. @mzegla

File: docs/parameters.md [hunk 2]
```diff
@@ -14,57 +14,10 @@
...
-Layout change is only supported to `NCHW` or `NHWC`. You can specify 2 forms of values:
-  * string - either `NCHW` or `NHWC`; applicable only for models with a single input tensor
-  * dictionary of strings - e.g. `{"input1":"NHWC", "input2":"NCHW", "output1":"NHWC"}`; allows to specify layout for multiple inputs and outputs by name.
-
-After the model layout is changed, the requests must match the new updated shape in order NHWC instead of NCHW. For NCHW inputs it should be: `(batch, channels, height, width)` but for NHWC this is: `(batch, height, width, channels)`.
-
-Changing layout is not supported for models with input names the same as output names.
-For the model included in DAG, layouts of subsequent nodes must match similarly to network shape and precision.
-
+| `"statefu"l` | `bool` | If set to true, model is loaded as stateful. |
```
Comment (dtrawins) [inline docs/parameters.md:17]:
```suggestion
| `"stateful"` | `bool` | If set to true, model is loaded as stateful. |
```

**Pull request merged**